src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/PromptManager.kt

```diff
@@ -39,6 +38,7 @@ class PromptManager(
     private val psiHelper: PsiHelper,
     private val caret: Int,
 ) {
+    // The classesToTest is empty when we work with the function outside the class
```

Comment:
Better to re-write as a Javadoc so that it's visible on hover:
```kotlin
/**
* The `classesToTest` is empty when we work with the function outside the class
*/
private val classesToTest: List<PsiClassWrapper>
```